mrp_lot_on_hand_first/views/mrp_production.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="mrp_production_form_view_inherit" model="ir.ui.view">
+        <field name="name">mrp.production.form.inherit</field>
+        <field name="model">mrp.production</field>
+        <field name="inherit_id" ref="mrp.mrp_production_form_view" />
+        <field name="arch" type="xml">
+            <field name="use_create_components_lots" position="after">
+                <field name="display_lots_on_hand_first" invisible="1" />
+            </field>
+            <xpath
+                expr="//field[@name='move_raw_ids']//field[@name='lot_ids']"
+                position="attributes"
+            >
+            <attribute
+                    name="context"
+                    operation="python_dict"
+                    key="name_search_qty_on_hand_first"
+                >parent.display_lots_on_hand_first</attribute>
```

Comment:
just being curious, i see also lot of times construction with update - like below, why `python_dict` then?
```
<attribute name="context" operation="update">
          {"some_key": some_value}
</attribute>
```

Reply:
It used to be so in version 14.0, but you're right that it has to use update in 15.0